users/users.go

```diff
@@ -58,21 +58,19 @@ type average struct {
 }
 
 // CollectMetrics returns values of desired metrics defined in mts
-func (users *Users) CollectMetrics(mts []plugin.PluginMetricType) ([]plugin.PluginMetricType, error) {
-	metrics := make([]plugin.PluginMetricType, len(mts))
+func (users *Users) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, error) {
+	metrics := make([]plugin.MetricType, len(mts))
 
 	err := users.getUsersStats()
 	if err != nil {
 		return nil, err
 	}
 
-	hostname, _ := os.Hostname()
 	for i, m := range mts {
-		if v, ok := users.data[parseNamespace(m.Namespace())]; ok {
-			metrics[i] = plugin.PluginMetricType{
+		if v, ok := users.data[parseNamespace(m.Namespace().Strings())]; ok {
+			metrics[i] = plugin.MetricType{
 				Namespace_: m.Namespace(),
 				Data_:      v,
-				Source_:    hostname,
```

Comment:
as you removed the source, is it good to add the tag or Snap takes care of it already?